### ISSUE TITLE
Draft: Clarify specific translated string in `draftguitools/gui_selectplane.py` so that it's less ambiguous

### ISSUE THREAD

**Issue by luzpaz** [FreeCAD/FreeCAD#9406]
https://github.com/FreeCAD/FreeCAD/blob/25157275f07075ccd40873837f9b0f48aef28c5a/src/Mod/Draft/draftguitools/gui_selectplane.py#L523

Most likely in context meant to be 'Direction'. But some translators might confuse with 'Directory'.

**Comment by yorikvanhavre** [comment]
This text is written on the toolbar button. It needs to be kept as short as possible to not make the button too long or have truncated text. However maybe there is an unicode icon that could represent a direction better? Maybe ↗ ?

**Comment by d1nuc0m** [comment]
Is it possible to add a comment visible to translators? Something like "Dir here means Direction, not Directory. Also shorten the translation because of available space in GUI"

**Comment by luzpaz** [comment]
We can use the 3rd field: 
`vdir = " " + translate("draft", "Dir", "Dir here means Direction, not Directory. Also shorten the translation because of available space in GUI") + ": " + vdir`

**Comment by d1nuc0m** [comment]
> We can use the 3rd field: `vdir = " " + translate("draft", "Dir", "Dir here means Direction, not Directory. Also shorten the translation because of available space in GUI") + ": " + vdir`

I don't know the details of the translation system, but if it works... why not?

**Comment by chennes** [comment]
Yeah, using the comment string is a good idea here. Whether translators actually notice the comment is an open question, of course :).
